course_flow/models.py

```diff
@@ -18,7 +19,11 @@ class Column(models.Model):
     created_on = models.DateTimeField(auto_now_add=True)
     last_modified = models.DateTimeField(auto_now=True)
     
-    default_activity_columns = ["ooci", "ooc", "ici", "ics"]
+    default_columns = {
```

Comment:
add a `PositiveIntegerField` here and copy the format from the `Node` model. Not urgent, I know you're not done with this model.

Reply:
I'd actually ideally like to store stuff like this in a separate file. I realize that this might not be the standard convention, but when I worked on large n-body codes in which there would be a large number of default settings, constants, etc, we would keep everything like this organized in a constants.x file and import what we needed from it. Helps keep things DRY. Think that could work here? I'd probably want a constants.py file and a constants.js file for back and front end respectively.

Reply:
Yeah that sounds good. But shouldn't we still have some sort of choice field to register what type of column it is?

Reply:
Yep, good point. I think I had misunderstood what you meant.